wurst/_wurst/Orders.wurst

```diff
@@ -0,0 +1,379 @@
+package Orders
+import NoWurst
+
+/** Class that contains every known order in game that has no string counterpart */
+public class SpecialOrders
+	static constant tomeOfAttack			 = 852259
+	static constant wandOfIllusion			 = 852274
+
+
+
+/*
+	Source: cj_order.j v 0.12
+		( Orders Define Library )
+
+    This was taken from cJass standard library
+	2009 Shadow Daemon \\ cjass.xgm.ru
+*/
+/** Class that contains every known order in game. */
+public class Orders extends SpecialOrders
```

Comment:
the indentation is still mixed: https://i.imgur.com/eNy9ZIG.png
See yourself https://github.com/wurstscript/WurstStdlib2/pull/21/files